Developers/AniketWayal/.gitignore

```diff
@@ -0,0 +1 @@
+/.metadata/
```

Comment:
Let have a one .gitignore in folder like 
Developers/AniketWayal/.gitignore

Dont add into remain folders